driver-core/src/main/com/mongodb/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -113,7 +113,8 @@ public void onResult(final List<RawBsonDocument> result, final Throwable t) {
     }
 
     private void retryOperation(final AsyncBlock asyncBlock, final SingleResultCallback<List<RawBsonDocument>> callback) {
-        changeStreamOperation.resumeAfter(resumeToken).executeAsync(binding, new SingleResultCallback<AsyncBatchCursor<T>>() {
+        changeStreamOperation.resumeAfter(resumeToken).startAtOperationTime(null)
```

Comment:
When retrying the startAtOperationTime needs to be null.

Reply:
This doesn't seem to handle the conditional in the spec where

> the ChangeStream has not received any changes, and resumeAfter is not specified, and the server version is >= 4.0

See https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#resume-process

Reply:
Done